src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -1,4 +1,59 @@
 featurizers:
+  # ----------------------------------------------------------------------
+  - name: BackwardFillImputerFeaturizer
+    estimator_name: BackwardFillImputerEstimator

```

Comment:
This featurizer wont always return a row and needs flush, so `has_dynamic_output` is needed.

Reply:
> This featurizer wont always return a row and needs flush, so `has_dynamic_output` is needed.

That is specified below (line 31), but I will move it up so it is more visible.